spec/contacts/publish_a_contact_spec.rb

```diff
@@ -22,4 +22,12 @@ def then_i_can_view_it_on_gov_uk
     expect_url_matches_live_gov_uk
     expect_rendering_application("government-frontend")
   end
+
+  def and_i_can_view_it_on_finder
+    contact_url = Pathname.new(current_url).parent.to_s
```

Comment:
I imagine here it might be a little more end-to-end like if we click the breadcrumb to go back. Not sure if that is hard to match or not though. What do you think?

Reply:
Based on the screenshots we've seen the breadcrumb isn't being rendered in test and it doesn't show locally either. Not sure if we're missing something that does the nav

Reply:
I'd presume this line is being hit:

https://github.com/alphagov/government-frontend/blob/master/app/presenters/contact_presenter.rb#L116

I'm guessing we have an empty organisation there because of the way we are seeding the organisation in Whitehall?